lib/open_pantry/web/templates/food_selection/index.html.eex

```diff
@@ -52,7 +52,7 @@
           </span>
           <span class="hidden js-credit-type"><%= type_id %></span>
         </span>
-      <%= end %>
```

Comment:
hah thank you I have this warning fixed locally and keep forgetting to commit somewhere.